### PR TITLE
Implement error handling

### LINKED ISSUE #63: Implement more informative and stylistically consistent error messages
Recently the Toast component was used to display error messages instead of browser alerts for the Post/User/Comment error handling effort. However, this upgrade is not universally applied and also does not account for any errors besides image upload errors. Therefore I'm interested in making Toast use consistent as well as more specific error messages.

### PR DESCRIPTION
**Pull request by averyfairbanks**
Closes #63

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
   },
   "devDependencies": {
     "@testing-library/jest-dom": "^5.12.0",
-    "@testing-library/react": "^11.2.7",
+    "@testing-library/react": "^12.0.0",
```
Comment (forrestwilkins) [inline package.json]: Why are we updating dependencies here?

**Comment by forrestwilkins** [comment]
Looking good! Can we resolve linting errors for:
- `resolvers/post.ts`
- `Posts/Form.tsx`
- `_App/Layout.tsx`

You might able to resolve them all with just `yarn lint --fix`

File: package.json (hunk 1 above)
Comment (averyfairbanks) [reply package.json]: My IDE will simply not behave, this goes for dependencies and linting errors. Will handle now!

**Review by averyfairbanks — COMMENTED** [review]
This should solve linting errors and fix erroneously updated dependencies, async support for try catch blocks needs to be added: [Async Err Catching REF](https://softwareengineering.stackexchange.com/questions/144326/try-catch-in-javascript-isnt-it-a-good-practice/302656#302656)

File: src/locales/en.ts [hunk 2]
```diff
@@ -29,6 +29,18 @@ const en = {
...
+      postUpdateError: () => "Post could not be edited",
+    },
+    comments: {
+      commentCreationError: () => "Unable to post comment",
+      commentUpdateError: () => "Unable to update comment",
+    },
+    users: {
+      userCreationError: () => "There was an error creating your account",
+      userUpdateError: () => "There was an error updating your account",
+    },
```
Comment (forrestwilkins) [inline src/locales/en.ts]: Can these be moved to their respective models, so `errors.posts` becomes `posts.errors` and so on?

File: package.json (hunk 1 above)
Comment (forrestwilkins) [reply package.json]: Much better! Can we also revert any changes to yarn.lock?

Comment (averyfairbanks) [reply package.json]: > 
> 
> Much better! Can we also revert any changes to yarn.lock?


Resolved in 8205773

File: src/locales/en.ts (hunk 2 above)
Comment (averyfairbanks) [reply src/locales/en.ts]: Please review 05b051c and let me know what you think!

**Comment by forrestwilkins** [comment]
Work started in this PR has been merged into main with #93.

**Pull request closed without merging**